webui/react/src/pages/Settings/SettingsAccount.tsx

```diff
@@ -57,6 +57,7 @@ const SettingsAccount: React.FC = () => {
       <div className={css.row}>
         <label>Display Name</label>
         <InlineEditor
+          pattern={new RegExp('^[a-zA-Z]+[a-zA-Z0-9]*$')}
```

Comment:
idk if there is a significant performance difference, but could also be written as:
```
new RegExp('^[a-z]+[a-z0-9]*$', 'i')
```
OR (this one is succinct but may not be as typescript friendly? not sure)
```
/^[a-z]+[a-z0-9]*$/i
```

Reply:
That's a good suggestion. Enabling insensitivity improves the performance a lot in my experience